### ISSUE TITLE
doesn't work with yarn 4 and pnp 

### ISSUE THREAD

**Issue by adrian-gierakowski** [coderaiser/putout#214]
due to undeclared dependencies 

```
> yarn --version 
4.1.1

> cat .yarnrc.yml
nodeLinker: pnp

> yarn add -D putout@npm:35.37.0

> yarn putout src
🐊 @putout/formatter-progress-bar: @putout/engine-reporter tried to access @putout/formatter-progress-bar, but it isn't declared in its dependencies; this makes the require call ambiguous and unsound.

Required package: @putout/formatter-progress-bar (via "@putout/formatter-progress-bar/package.json")
Required by: @putout/engine-reporter@virtual:24198d8dec95d3ce8be035cf36d5bd5bdb578443242dcefdc571e238f3ca6194f33f01795bd954621471d0ffcf118db407fcabdeeaf53998567994c93a0b4741#npm:1.0.2 (via /code/rhinofi/rhino-core/.yarn/__virtual__/@putout-engine-reporter-virtual-53a01f8625/4/home/adrian/.yarn/berry/cache/@putout-engine-reporter-npm-1.0.2-8960bc2c33-10c0.zip/node_modules/@putout/engine-reporter/lib/simple-import.cjs)
```

**Comment by coderaiser** [comment]
`@putout/engine-reporter` is dependency of `putout`, `@putout/formatter-progress-bar` is dependency of `putout`. Do you have any ideas how to handle this?

**Comment by coderaiser** [comment]
Just fixed in **Putout** [v35.37.1](https://github.com/coderaiser/putout/releases/tag/v35.37.1) 🎉 . Is it works for you?
